packages/material-ui-system/src/GlobalCss/displays.js

```diff
@@ -0,0 +1,115 @@
+import combineWithBreakpoints from './combineWithBreakpoints';
+
+const displaysProps = {
+  'd-inline': { display: 'inline !important' },
```

Comment:
I strongly think that we should remove all usage of `!important`. It prevents developers to create abstractions with these utility class names. Instead, we need to figure out what's going wrong. Either it's because the import order is wrong (and we need to document it) or because the component increases specificity and we need to look into how to flatten it.